src/docs/product/issues/issue-details/performance-issues/regressions.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Function Regression"
+sidebar_order: 40
+description: "Learn more about Function Regression Issue and how to diagnose and fix them."
+---
+
+Function Regression issues capture a generic class of problems where the duration of a function has increased over time. This can lead to degraded performance in your application. Sentry attempts to monitor some of your most common functions out of the box and report any possible regressions under the Function Regression issue type.
+
+## Detection Criteria
+
+[Profiling](/product/profiling) must be enabled for Sentry to detect Function Regression issues. Once set up, Sentry will periodically check the 95th percentile function duration of the most common application functions in your project. When a significant change in the function duration for a function is detected and it has sustained for some time, a regression issue is reported.
+
+## Charts
+
+The chart shows a window of up to 14 days before and after where the regression was identified. Here you can see the p95 function duration as well as the function throughput.
+
+A common cause of regressions is an increase in throughput of a function in the code path that contains a mutex which is now causing lock contention.
```

Comment:
```suggestion
A common cause of regressions is an increase in the throughput of a function. For example, when the throughput increases in a code path that contains a mutex, this can cause lock contention.
```
found the original kinda hard to read with all the of, in, that, which... hope this suggestion is still accurate